### ISSUE TITLE
Make swan's launcher be able to stay on top of the taskbar.

### ISSUE THREAD

**Issue by NeeharVijay** [starlight/swan-desktop#56]
When you put swan's launcher on taskbar then you press the taskbar, it goes underneath it. (unlock taskbar and drag to other side to recover to recover)
It's a pity because it was a good place to put it.

**Comment by isaacwhanson** [comment]
I don't think that is something that can be corrected, unfortunately.  If someone figures out a way, would be glad to hear it.
